modules/ossm-security-auth-policy.adoc

```diff
@@ -0,0 +1,180 @@
+////
+Module included in the following assemblies:
+-service_mesh/v2x/ossm-security.adoc
+////
+
+[id="ossm-vs-istio_{context}"]
+= Role Based Access Control (RBAC)
+
+Role-based access control (RBAC) objects determine whether a user or service is allowed to perform a given action within a project. You can define mesh-, namespace-, and workload-wide access control for your workloads in the mesh. 
+
+To configure RBAC, create an `AuthorizationPolicy` resource in the namespace for which you are configuring access. If you are configuring mesh-wide access, use `istio-system`. The namespace must match your `metadata.namespace` field.
+
+For example, with RBAC, you can create policies that:
+
+* Configure intra-project communication.
+* Allow or deny full access to all workloads in the default namespace.
+* Configure ingress gateway access.
+* Require a token for access,
+
+An authorization policy includes a selector, an action, and a list of rules:
+
+* The `selector` field specifies the target of the policy.
+* The `action` field specifies whether to allow or deny the request.
+* The `rules` field specifies when to trigger the action.
+** The `from` field specifies constraints on the request origin.
+** The `to` field specifies constraints on request target and parameters.
+** The `when` field specifies additional conditions that to apply the rule.
+
+.Procedure
+
+1. Construct your `AuthorizationPolicy` resource. The following example shows a resource that updates the ingress-policy `AuthorizationPolicy` to deny an IP address from accessing the ingress gateway.
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: ingress-policy
+  namespace: istio-system
+spec:
+  selector:
+    matchLabels:
+      app: istio-ingressgateway
+  action: DENY
+  rules:
+  - from:
+    - source:
+      ipBlocks: ["1.2.3.4"]
+----
++
+2. Run the following command after you write your resource to create your resource in your namespace. The namespace must match your `metadata.namespace` field.
```

Comment:
I think this is clear. In the previous paragraph, you can configure mesh-wide access. To do that, you can add this resource to the same namespace where you create your control plane. To do it in your namespace, you create it there. This is open-ended. I don't think I should be more specific at this point in the topic.

I removed the namespace much match because it's included again later in the procedure where it's more helpful.